src/Control/Bind.purs

```diff
@@ -73,11 +73,11 @@ foreign import arrayBind :: forall a b. Array a -> (a -> Array b) -> Array b
 -- |
 -- | An example is the `Unit` type, since there is only one
 -- | possible value which can be returned.
-class Discard a where
-  discard :: forall f b. Bind f => f a -> (a -> f b) -> f b
+class BindDiscard a where
+  bind_ :: forall f b. Bind f => f a -> (a -> f b) -> f b
```

Comment:
Honestly, I prefer `discard` the more I think about it :)

`ignore` or `ignoring` is another option.